Sources/Shared/Notifications/LocalPush/PushProviderConfiguration.swift

```diff
@@ -1,8 +1,8 @@
 public struct PushProviderConfiguration: Codable {
-    public static var providerConfigurationKey = "ha_configurations"
+    public static let providerConfigurationKey = "ha_configurations"
 
-    public var serverIdentifier: Identifier<Server>
-    public var settingsKey: String
```

Comment:
I think that cases like this - where it's a data type - shouldn't force the non-static members to be `let` - that's already done by the mutability state of the struct. It's not _in and of itself wrong_ to copy a struct and mutate the value for some reason, as long as the values are valid state changes -- which the type can guard if it wants.

Reply:
Since there is no real impact now, I will merge this PR as-is, we can always change it to var on demand of course when needed to actually do something with the struct.